install.json

```diff
@@ -72,14 +72,30 @@
         },
         "default": "dark",
         "title": "Text shadow color",
-        "order": 4,
+        "order": 3,
         "type": "string"
       },
 
+      "redirect": {
```

Comment:
Additionally, the current behavior causes a refresh in the preview since a handler is not specified. 

Consult app.js for an example. Or poke me on Slack :)